test/blocks/featured-article/mocks/article.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
I like how you've rewritten the tests! Very cool 
Just a little thing though, you're not using this file anymore. If you're not testing with it I'd delete